content/applications/hr/appraisals/appraisal_analysis.rst

```diff
@@ -0,0 +1,109 @@
+of the report. The options to display are :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, and
+:guilabel:`Year`. Use the arrows to move forward or backward in time.
+
+At any point, click the :guilabel:`Today` button to have the Gantt view include today's date in the
+view.
+
+To view the details of nay appraisal, click on any appraisal. A pop-over appears, displaying the
+due date for the appraisal. To view more details, click the :guilabel:`View` button and further
+details appear in a pop-up window.
+
+The report can have other :ref:`filters <search/filters>` and :ref:`groupings <search/group>` set in
+the search bar at the top.
+
+.. image:: appraisal_analysis/analysis.png
+   :align: center
+   :alt: A report showing all the appraisals for the Appraisal Analysis report.
+
+.. _appraisals/group-status:
+
+Group by status
+===============
+
+When a company has a large number of employees, the default :guilabel:`Appraisal Analysis` report
+may display too much information to view easily. In this scenario, viewing the data by status can be
+beneficial.
+
+First, remove the default :icon:`oi-group` :guilabel:`Department` grouping. Next, click the
+:icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the far-right of the search bar. Click
+:guilabel:`Status` in the :icon:`oi-group` :guilabel:`Group By` column. Click off of the drop-down
+menu to close it.
+
+All the appraisals are now organized by status, in the following order: :guilabel:`Cancelled`,
+:guilabel:`Done`, :guilabel:`To Start`, and :guilabel:`Appraisal Sent`.
+
+This view makes it easy to see which appraisals need to be completed, and when, as well as which
+appraisals still need to be confirmed.
+
+.. image:: appraisal_analysis/by-status.png
+   :align: center
+   :alt: A report showing all the appraisals, grouped by status.
+
+Use case: view only your appraisals
+===================================
+
+When viewing the :guilabel:`Appraisal Analysis` report, it can be time-saving to only view the
+appraisals that the signed-in user is responsible for, and hide the rest.
+
+To view only this data, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the
+far-right of the search bar, revealing a drop-down menu. Click :guilabel:`Add Custom Filter` at the
+bottom of the :icon:`fa-filter` :guilabel:`Filters` column, and a :guilabel:`Add Cusomt Filter`
+pop-up window appears.
+
+Click into the first field, and a popover appears with a variety of options. Click the
+:icon:`fa-chevron-right` :guilabel:`(right arrow)` icon after the word :guilabel:`Employee`, and
+scroll down and click on :guilabel:`Manager`. Click away to close the popover. Change the middle
+drop-down to :guilabel:`=` :guilabel:`(equal)`. Last, click into the third drop-down and select
+the desired user from the list. When all the fields are set, click :guilabel:`Add`.
+
+.. image:: appraisal_analysis/custom.png
```

Comment:
this image a little stretched out, vertically. Let's try uploading this image again (:

![image](https://github.com/user-attachments/assets/c895c5ab-b4f6-4602-a458-c371248ac97a)

Reply:
Agree- I took this photo over 5 times, and it kept doing something wonky so I thought, I'll redo this tomorrow but first,  let's get this doc reviewed  =D

Reply:
OK, I FINALLY got it to work! I had to take a full-sized image, then change the dimensions in ANOTHER app since it kept getting messed up with Nimbus... OMG that took forever =D